packages/sources/svelte/src/components/structure/VtmnList/test/VtmnListWithSlots.svelte

```diff
@@ -0,0 +1,13 @@
+<script>
+  import VtmnList from '../VtmnList.svelte';
+  import VtmnListItem from '../../VtmnListItem/VtmnListItem.svelte';
+</script>
+
+<VtmnList {...$$restProps}>
+  <VtmnListItem index="1">
+    <svelte:fragment slot="text">Item 1</svelte:fragment>
+  </VtmnListItem>
+  <VtmnListItem index="2">
+    <svelte:fragment slot="text">Item 2</svelte:fragment>
+  </VtmnListItem>
```

Comment:
Why is there a z-index prop here ? Is it necessary ?

Reply:
Here the `VtmnListiItem` needs index for tabIndex, but yes, I will fix it to tab-index